### ISSUE TITLE
mute NG6002 when there are other issues as well

### ISSUE THREAD

**Issue by vmachacek** [angular/angular#58113]
### Is this a bug report or a feature request?

- [ ] Bug Report
- [x] Feature Request

### Description [Feature Request only]

When Im working on project with 30+ modules it can happen there is a lot of NG6002 errors which are not root cause of the issue. In such case I have to scroll thru the command line to locate compile type issue which will lead to the resolution of the issue

### Proposed solution [Feature Request only]

Write NG6002 to console in bulk or grey color or hide them completely (probably not a good idea) when other errors are present

### Alternatives considered [Feature Request only]

scrolling thru 200 lines of command line output

**Comment by vmachacek** [comment]
I was able to observe when there are only errors of type NG6002 and I restart `ng serve` they go away and different types of errors surface. That is a bug in my mind.

**Comment by JoostK** [comment]
> I was able to observe when there are only errors of type NG6002 and I restart `ng serve` they go away and different types of errors surface. That is a bug in my mind. 

Yeah, this sometimes happens, but I've never found a true reproduction.

As for this particular case, could you please share a repro where the output is excessive in terms of non-root cause errors?

**Comment by vmachacek** [comment]
The original "Feature request" reproduction conditions might be related to the "bug" side of things. I'm thinking import mechanism starts to fail due to compile error and later wont go away as CLI wont detect the error is gone. This is total shot in a dark and probably rather dumb.   As my project is large I wont think I will be able to share isolated reproduction steps with fresh repository. If that's what you are after.

**Comment by JoostK** [comment]
> I'm thinking import mechanism starts to fail due to compile error and later wont go away as CLI wont detect the error is gone

It is likely something like this, indeed, but the exact way this happens is unknown.

> As my project is large I wont think I will be able to share isolated reproduction steps with fresh repository. If that's what you are after.

We do indeed need a reproduction, ideally isolated but more importantly is that it reproduces consistently.

**Comment by pkozlowski-opensource** [comment]
Thanks for reporting this issue. However, you didn't provide sufficient information for us to understand and reproduce the problem. Please check out [our submission guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-an-issue) to understand why we can't act on issues that are lacking important information.

If the problem still exists in your application, please [open a new issue](https://github.com/angular/angular/issues/new/choose) and follow the instructions in the issue template.
